code/datums/mutantraces.dm

```diff
@@ -2019,6 +2064,17 @@
 			if ("milk")
 				if (mob.emote_check(voluntary))
 					.= release_milk()
+			if ("stamp")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> stamps their hoove!", "<i>stamps their hoove!</i>")
```

Comment:
```suggestion
					. = list("<B>[mob]</B> stamps their hooves!", "<i>stamps their hooves!</i>")
```

Reply:
I am very used to using they as opposed to gendered pronouns so it didn't really come into my mind, I can change it if its preferred by others.

To answer the list question, I would prefer that but I couldn't figure out how to handle that and have it look nice

Reply:
change it
singular they is **BIG CRINGE™**